homeassistant/components/comfoconnect/__init__.py

```diff
@@ -102,15 +155,15 @@ def __init__(self, hass, bridge, name, token, friendly_name, pin):
         )
         self.comfoconnect.callback_sensor = self.sensor_callback
 
-    def connect(self):
+    async def connect(self):
         """Connect with the bridge."""
         _LOGGER.debug("Connecting with bridge")
-        self.comfoconnect.connect(True)
+        await self.hass.async_add_executor_job(self.comfoconnect.connect, True)
```

Comment:
Why make this an async method if you are passing it as an executor job anyways?
Just keep it a sync method in that case.